core/xmake.lua

```diff
@@ -50,8 +50,8 @@ end
 
 -- for mode coverage
 if is_mode("coverage") then
-    add_cxflags("-coverage")
-    add_ldflags("-coverage")
+    add_cxflags("-coverage", "-fprofile-arcs", "-ftest-coverage")
+    add_ldflags("-coverage", "-fprofile-arcs", "-ftest-coverage")
```

Comment:
It doesn't seem to be able to generate '*.gcno' and '*.gcda' files for old version of gcc with flags `-coverage`. But `"-fprofile-arcs", "-ftest-coverage"` works.